thirdparty/basis_universal/encoder/3rdparty/android_astc_decomp.cpp

```diff
@@ -837,7 +837,7 @@ void decodeISETritBlock (ISEDecodedResult* dst, int numValues, BitAccessStream&
     deUint32 T7     = data.getNext(1);
 
 #ifndef __EMSCRIPTEN__
-#ifdef __GNUC__
+#if defined(__GNUC__) && !defined(__clang__)
```

Comment:
I'm surprised that it's not a problem upstream, they must support compiling with Clang.

It seems like the `#ifndef __EMSCRIPTEN__` here is meant to prevent adding these pragma too, so it could be dropped (and the change PR'ed upstream) as `__clang__` is defined for Emscripten too.